recipe/meta.yaml

```diff
@@ -7,10 +7,12 @@ package:
 source:
   url: https://github.com/pypa/virtualenv/archive/{{ version }}.tar.gz
   sha256: 74eb96b4e1eb08fbe37b5c6fbad9c978e1c9461013378f5df7637c2d8eccb18e
+  patches:
+    - 0001-Mach-O-Copy-any-dynamic-libraries-and-symlinks-to-th.patch
+    - 0002-ELF-Copy-any-dynamic-libraries-and-symlinks-to-the-c.patch
 
 build:
-  noarch: python
```

Comment:
@ocefpaf I think you removed this line by mistake.

Reply:
Yep. I used GH interface and that is easy to make mistakes.